### ISSUE TITLE
Is it possible to stream from phone to laptop?

### ISSUE THREAD

**Issue by bitflame** [hypeapps/Endoscope#37]
Hello 
I want to send traffic to laptop or Internet server from the phone. Is it possible?

**Comment by UsuarioAvanzado** [comment]
Yes, you can use any player that supports RTSP protocol streaming, such as VLC or MPV.

I use MPV by typing in the console:

`mpv rtsp://IP-ADDRESS:PORT --rtsp-transport=udp`


For example: 

`mpv rtsp://192.168.0.20:8086 --rtsp-transport=udp`

**Comment by tuxflo** [comment]
This is what the setting looks like in VLC:
![image](https://user-images.githubusercontent.com/2816893/166219547-3ede2417-67fe-4bd9-be2b-06da5de30c1a.png)

I think this issue could be closed then.
BTW: Thanks for the app, it's awesome

**Comment by psa-jforestier** [comment]
I'm trying to expose my Android device on the internet, to be able to stream the video to my laptop (testing purpose only). It works locally with VLC when my PC is on the same network as my Mobile, if I use the local network address (rtsp://192.168.1.100:8086 for example).
On my internet router, I did a NAT rule to forward TCP+UDP on port 8086 to the Android device. The router is publicly available at "myrouter.mydomain.com".
But with VLC, it doesnt work if I use the public hostname. It display a black screen, without any particular message.
I'm sure the public ports are open and traffic is forwared (I can connect with Curl or Putty), 
I do not know the RTSP protocol, maybe there is something like the http "host" headers, and the mobile app request the client to connect via an IP address and not a public hostname ?
